c2corg_api/models/user.py

```diff
@@ -103,6 +103,10 @@ class User(Base):
     feed_filter_activities = Column(
         ArrayOfEnum(enums.activity_type), nullable=False, server_default='{}')
 
+    # only show updates from followed users in the homepage feed
+    feed_followed_only = Column(
```

Comment:
No need to implement it now, but do you think it would be possible to temporarly switch off the filters (and subscriptions) without dropping all the activities and areas filters in the user prefs UI?

Some kind of button that we would show on top of the homepage feed, a bit like the bulb in v5:
![sans titre](https://cloud.githubusercontent.com/assets/1192331/19196342/74dec558-8cb5-11e6-92f8-4ca2dc7d51ad.png)